system/jlib/jmutex.hpp

```diff
@@ -290,7 +290,7 @@ class CriticalSection
     inline ~CriticalSection()
     {
 #ifdef _ASSERT_LOCK_SUPPORT
-        assert(owner==0 && depth==0);
+//        assert(owner==0 && depth==0);
```

Comment:
left in by mistake?

Reply:
Forgot about this. This is an issue that I need to fix.